src/screens/ProfileScreen/index.js

```diff
@@ -629,23 +629,25 @@ const ProfileScreen = ({route}) => {
   };
 
   const onHeaderOptionClicked = (post) => {
+    console.log('post', post);
     setSelectedPostForOption(post);
     setIsOptionModalOpen(true);
   };
 
-  const removePostByIdFromContext = () => {
-    const deletedIndex = feeds?.findIndex((find) => selectedPostForOption?.id === find?.id);
+  const removePostByIdFromContext = (post) => {
+    const deletedIndex = feeds?.findIndex((find) => post?.id === find?.id);
     const newData = [...feeds];
     newData?.splice(deletedIndex, 1);
     setMyProfileFeed(newData, myProfileDispatch);
   };
 
-  const onDeletePost = async () => {
-    removePostByIdFromContext();
+  const onDeletePost = async (item) => {
+    removePostByIdFromContext(item);
 
     let response;
-    if (isProfileTabSigned) response = await deletePost(selectedPostForOption?.id);
-    else response = await deleteAnonymousPost(selectedPostForOption?.id);
+    console.log('selected post', item?.id);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, it's not a good practice to leave console.log statements in the production code. It can expose potentially sensitive user data and make debugging harder.
```diff
- console.log('selected post', item?.id);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->